docs/docs/faq.mdx

```diff
@@ -183,11 +183,11 @@ export VIRTUAL_ENV_DISABLE_PROMPT=1
 ### Text decoration styles like bold and dimmed don't work
 The text decoration styles are based on your terminal's capabilities.
 
-A quick way to check if your terminal supports the specific style escape sequence is to take a look at the `terminfo` database, on Linux.
+A quick way to check if your terminal supports a specific style escape sequence is to take a look at the `terminfo` database, on Linux.
 Refer [this page on ArchWiki](https://wiki.archlinux.org/title/Bash/Prompt_customization#Terminfo_escape_sequences).
 
 If you are on Windows, use Windows Terminal. It closely copies the `xterm-256color` capabilities.
-See the [PowerShell docs] and [this GitHub comment](https://github.com/microsoft/terminal/issues/6045#issuecomment-631743728).
+See the [PowerShell docs](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_ansi_terminals?view=powershell-7.2) and [this GitHub comment](https://github.com/microsoft/terminal/issues/6045#issuecomment-631743728).
```

Comment:
Add the links below like the others:

```markdown
[PowerShell docs][ps-docs] and and [this GitHub comment][gh-comment]

...
[ps-docs]: https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_ansi_terminals?view=powershell-7.2
[gh-comment]: https://github.com/microsoft/terminal/issues/6045#issuecomment-631743728
```

Reply:
Same for arch above, missed that initially.

Reply:
Pushed a new commit. Sorry for the inconsistent styling.